SwiftGit2Tests/RepositorySpec.swift

```diff
@@ -600,6 +600,33 @@ class RepositorySpec: QuickSpec {
 				expect(repo.HEAD().value?.longName).to(equal("HEAD"))
 			}
 		}
+
+		describe("Repository.allCommits(in:)") {
+			it("should return all (9) commits") {
+				let repo = Fixtures.simpleRepository
+				let branches = repo.localBranches().value!
+				let expectedCount = 9
+				let expectedMessages: [String] = [
+					"List branches in README\n",
+					"Create a README\n",
+					"Merge branch 'alphabetize'\n",
+					"Alphabetize branches\n",
+					"List new branches\n",
+					"List branches in README\n",
+					"Create a README\n",
+					"List branches in README\n",
+					"Create a README\n"
+				]
+				var commitMessages: [String] = []
+				for branch in branches {
+					while let commit = repo.commits(in: branch).next() {
```

Comment:
We'll want this to be `for commit in repo.commits(in: branch) {` when the map above goes away.